From d8f98dd9f2b437cca5954b38cb48d9c21733f67d Mon Sep 17 00:00:00 2001
From: James Hilliard <james.hilliard1@gmail.com>
Date: Thu, 24 Nov 2022 14:29:06 -0700
Subject: [PATCH] Fix FindFlatBuffers cmake file

Capitalization needs to match for system cmake override to work:
https://github.com/google/flatbuffers/blob/v22.11.23/CMake/FindFlatBuffers.cmake

Signed-off-by: James Hilliard <james.hilliard1@gmail.com>
Upstream: https://github.com/tensorflow/tensorflow/commit/d8f98dd9f2b437cca5954b38cb48d9c21733f67d
---
 tensorflow/lite/CMakeLists.txt                                  | 2 +-
 tensorflow/lite/g3doc/guide/build_cmake.md                      | 2 +-
 .../modules/{FindFlatbuffers.cmake => FindFlatBuffers.cmake}    | 0
 3 files changed, 2 insertions(+), 2 deletions(-)
 rename tensorflow/lite/tools/cmake/modules/{FindFlatbuffers.cmake => FindFlatBuffers.cmake} (100%)

diff --git a/tensorflow/lite/CMakeLists.txt b/tensorflow/lite/CMakeLists.txt
index f9c30d6a046..c2ee9edfb61 100644
--- a/tensorflow/lite/CMakeLists.txt
+++ b/tensorflow/lite/CMakeLists.txt
@@ -143,7 +143,7 @@ find_package(absl REQUIRED)
 find_package(Eigen3 REQUIRED)
 find_package(farmhash REQUIRED)
 find_package(fft2d REQUIRED)
-find_package(Flatbuffers REQUIRED)
+find_package(FlatBuffers REQUIRED)
 find_package(gemmlowp REQUIRED)
 find_package(NEON_2_SSE REQUIRED)
 find_package(cpuinfo REQUIRED)  #CPUINFO is used by XNNPACK and RUY library
diff --git a/tensorflow/lite/g3doc/guide/build_cmake.md b/tensorflow/lite/g3doc/guide/build_cmake.md
index 6950a365bb8..9cb7ebac326 100644
--- a/tensorflow/lite/g3doc/guide/build_cmake.md
+++ b/tensorflow/lite/g3doc/guide/build_cmake.md
@@ -84,7 +84,7 @@ cmake ../tensorflow_src/tensorflow/lite -DTFLITE_ENABLE_INSTALL=ON \
   -DSYSTEM_FARMHASH=ON \
   -Dabsl_DIR=<install path>/lib/cmake/absl \
   -DEigen3_DIR=<install path>/share/eigen3/cmake \
-  -DFlatbuffers_DIR=<install path>/lib/cmake/flatbuffers \
+  -DFlatBuffers_DIR=<install path>/lib/cmake/flatbuffers \
   -DNEON_2_SSE_DIR=<install path>/lib/cmake/NEON_2_SSE \
   -Dcpuinfo_DIR=<install path>/share/cpuinfo \
   -Druy_DIR=<install path>/lib/cmake/ruy
diff --git a/tensorflow/lite/tools/cmake/modules/FindFlatbuffers.cmake b/tensorflow/lite/tools/cmake/modules/FindFlatBuffers.cmake
similarity index 100%
rename from tensorflow/lite/tools/cmake/modules/FindFlatbuffers.cmake
rename to tensorflow/lite/tools/cmake/modules/FindFlatBuffers.cmake
-- 
2.34.1

